verification/src/compiler/version.rs

```diff
@@ -108,25 +108,25 @@ pub enum CompilerVersion {
 }
 
 impl CompilerVersion {
-    fn version(&self) -> &Version {
+    pub fn version(&self) -> &Version {
```

Comment:
convinient for tests

Reply:
> Ok, I thought about this and it seems like just my quirk, so think about this just as my thoughts, not a review comment.

I kinda dislike this functions, because instead of writing (for example)
```rust
let (semver, date, commit) = match ver {
  CompilerVersion::Nightly(v) => {
    (&v.version, Some(&date), &v.commit)
  }
  CompilerVersion::Release(v) => {
    (&v.version, None, &v.commit)
  }
}
```
you will probably write 
```rust
let semver = ver.version();
let date = ver.date();
let commit = ver.commit();
```
and you may ask "what's wrong with it, it's much clearer!", but in the second version there is 3 match clauses, instead of one, which, first of all, _blahblah performance blahblah_, but for me it's something about the semantics, like why do we need 6 matches instead of two.